influxdb_iox/tests/end_to_end_cases/querier.rs

```diff
@@ -124,6 +124,89 @@ async fn basic_no_ingester_connection() {
     .await
 }
 
+#[tokio::test]
+async fn query_after_persist_sees_new_files() {
+    // https://github.com/influxdata/influxdb_iox/issues/4634 added
+    // caching of tombstones and parquet files in the querier. This
+    // test ensures that a query issued after new parquet files is
+    // issued, they are correctly picked up
```

Comment:
I'm not sure what this comment is saying, maybe it's supposed to say:

```suggestion
    // test ensures that a query issued after new parquet files are
    // persisted correctly picks up the new parquet files
```

?